### DISCUSSION TITLE
Explain Copilot

### DISCUSSION THREAD

**Discussion by MidhiaQueiroz** [community/community#67430] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

I  have several app ideas that I will put to work. How and where can I find trustworthy professionals in writing the code for applications?

**Comment by mehulrajdev** [comment, marked as answer]
Online job boards: There are many online job boards where you can post your project and find developers who are interested in working on it. Some popular options include Upwork, Fiverr, and Toptal.
Networking: Talk to your friends, family, and colleagues to see if they know any developers who might be a good fit for your project. You can also attend industry events or meetups to meet new people.
